### ISSUE TITLE
Json Decode Error

### ISSUE THREAD

**Issue by Charan-Karthikeyan** [Uberi/speech_recognition#210]
Steps to reproduce
------------------
```python
import speech_recognition as sr
import requests
import json


from os import path
AUDIO_FILE = path.join(path.dirname(path.realpath(__file__)), "file.wav")

r = sr.Recognizer()
with sr.AudioFile(AUDIO_FILE) as source:
    audio = r.record(source) # read the entire audio file

GOOGLE_CLOUD_SPEECH_CREDENTIALS = r"/home/charan/Desktop/Speech/Speech-5156b41357e6.json"
try:
    # for testing purposes, we're just using the default API key
    # to use another API key, use `r.recognize_google(audio, key="GOOGLE_SPEECH_RECOGNITION_API_KEY")`
    # instead of `r.recognize_google(audio)`
    print("Google Speech Recognition thinks you said " + r.recognize_google_cloud(audio, credentials_json="GOOGLE_CLOUD_SPEECH_CREDENTIALS"))
except sr.UnknownValueError:
    print("Google Speech Recognition could not understand audio")
except sr.RequestError as e:
    print("Could not request results from Google Speech Recognition service; {0}".format(e))
    raise
```

Expected behaviour
------------------
The audio file must be converted to text using 

Actual behaviour
----------------

JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```



System information
------------------

(Delete all the statements that don't apply.)

My **system** is <Ubuntu 16.04 LTS x64>. (For example, "", "Windows 10 x64", or "macOS Sierra".)

My **Python version** is <3.6.>. (You can check this by running `python -V`.)

My **SpeechRecognition library version** is <3.6.0>.

**Comment by Uberi** [comment]
Hi @Charan-Karthikeyan,

Your example passes in the literal string `"GOOGLE_CLOUD_SPEECH_CREDENTIALS"`. Please refer to [the documentation](https://github.com/Uberi/speech_recognition/blob/9375607d2fc8c2b09b754d554eb44241686a6c41/reference/library-reference.rst#recognizer_instancerecognize_google_cloudaudio_data-credentials_json--none-language--en-us-preferred_phrases--none-show_all--false):

> The text content of this JSON file is specified by ``credentials_json``. If not specified, the library will try to automatically [find the default API credentials JSON file](https://developers.google.com/identity/protocols/application-default-credentials>).

In other words:

```python
import speech_recognition as sr
import requests
import json


from os import path
AUDIO_FILE = path.join(path.dirname(path.realpath(__file__)), "file.wav")

r = sr.Recognizer()
with sr.AudioFile(AUDIO_FILE) as source:
    audio = r.record(source) # read the entire audio file

with open(r"/home/charan/Desktop/Speech/Speech-5156b41357e6.json", "r") as f:
    credentials_json = f.read()
print("Google Speech Recognition thinks you said " + r.recognize_google_cloud(audio, credentials_json=credentials_json))
```
